index.html

```diff
@@ -173,6 +173,30 @@ <h2>Calls for implementation and feedback</h2>
 
 <p>When an addition is accepted at the “candidate” (stage 3) maturity level, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage and external feedback.
 
+<h2>Withholding Consensus for Advancement</h2>
+
+<p>During the discussion of a proposal, at any stage, any aspect may be discussed, including those outside of the scope of its current stage. A proposal’s advancement may not have consensus, and may be "blocked". A block is a forumated reason for why a proposal should not advance in it's present form. The committee does not have a concept of a rejected proposal.</p>
+
+<p>In withholding consensus for advancement, a reason should be stated either as a delay or as a block. If no reason is given, the objection is void for vagueness. In the case of a delay -- a delegate may need more time to consider a given aspect of a proposal. They may request additional time, or allow conditional advancement.</p>
+
+<p>Conditional advancement is time-limited, giving the person raising the block time to discuss with the champions and authors about their concerns. If a proposal has conditional advancement, an issue is opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.</p>
+
+<p>A Block needs to be well described and recorded within the committee, so that in future meetings and future proposals we can learn from failed proposals, and have a good record of why they failed.</p>
+
+<p>Regardless of stage, the following types of blocks require the committee to reach consensus in order for the block to be valid: Blocks related to an earlier stage, Blocks which have been discussed at length by the committee and were resolved, and insufficently stated blocks.</p>
```

Comment:
This is really confusing. I understand we are asking consensus for a reason that blocks a proposal from advancing. Some of the items are rather vague: "Blocks related to an earlier stage". I have some reference but only due to my experience at the committee, but it might need way more clarification for new comers.

The paragraph on like 190 might already solve the problem here.

I'd prefer if we rephrase this w/ something like "The committee might also request consensus for the reasons presented for blocking or delaying a proposal from stage advancement. Well stated reasons are expected to be objective, technical, and should provide an actionable plan whenever possible."

This suggested paragraph supports the following with connotation for well stated reasons.

Reply:
I don't think " objective" works -- people will either consider their positions to be objective, or will be presenting a very valid subjective view, i.e., a member industry position. I reworked that sentence a bit, let me know what you think.